### ISSUE TITLE
Logo Design For Bollib

### ISSUE THREAD

**Issue by zularizal** [LaunchMenu/Bollib#1]
Hi @sancarn , I want to make a logo for this project. I need some information about this project.

Can you explain a more detailed description of this project?
And what is the purpose of this project?

Thanks.

**Comment by TarVK** [comment]
Hey @zularizal ,

I shall be answering these questions, as I am the one that has mainly been writing code for it.

So basically Bollib will be an application framework, mainly designed for usage in open source projects, and with user customizability as a main priority.
Basically with bollib you can design an application as a bundle of separate 'plugins', where any of these plugins can easily be replaced with other versions. Other people can create these alternative versions, and the user can just install whichever one he/she likes. And easily customize small parts of the software this way.
At the same time, all these plugins will get their own location on the screen, but this location is once again customizable by the user. This works in a similar way to how you can move tabs/sections around in  for instance adobe after effects. 

So basically I think the keywords would be user customizable and plugins.
And just to be clear in case this description is confusing, Bollib is not an actual user product. Instead it is a tool with which user products can be made. LaunchMenu will be made using Bollib for instance, and hopefully other programs will follow in the future.

Thank you for considering making a logo for this too, I really like the one you made for LaunchMenu!

**Comment by zularizal** [comment]
Hi @TarVK ,  sorry for a long time. I have a lot of work here. But I have designed a logo for your project. This is the design logo for your project. I display it in one color. After you choose one or more, then I will give it the color you want. What do you think? Which do you like?
![rect4235](https://user-images.githubusercontent.com/38547278/56467890-0294a200-644f-11e9-8e2d-4f09ef6c8c71.png)

**Comment by TarVK** [comment]
Greetings @zularizal ,
No problem it took some time, in general there is no rush what so ever. We were quite curious what you would come up with though. However I must say I am unfortunately personally not in love with any of these. I don't want to ask too much of you, but if you would like to try something else I do have some feedback. If you don't want to try any adaptations because you have better things to do, that's totally understandable too! But as said, there is no rush at all, so if you don't have time right now but do later, that would definitely be appreciated as well.

So here is what I do like:
-A has a rather simple concise design
-B (potentially unintentionally) works quite well with our goal in my opinion, since it's basically two interlocking parts. And our system is about interlocking modules, sort of.

What I don't like:
-A and C seem to have some rather arbitrary curves, I would personally prefer something more 'predictable' 
-B is a little too tall in my opinion
-It would be cool if the curves used were similar to those from LaunchMenu's logo, given that LaunchMenu is built on top of the Bollib framework

I really don't want to be ungrateful, but none of these exactly do it for me. But I think alternative versions of either A or B might. Please let me know whether you will be able to make adaptations or not, and thanks for the work in any case!

-Tar van Krieken
